### ISSUE TITLE
Contradictory messaging

### ISSUE THREAD

**Issue by andreifoldes** [brainlife/ezbids#50]
Hello

When finalizing my bids dataset I receieve on top
```
All Done!
Please download the BIDS formatted data to your local computer
```

but if I scroll down I see
`failed to run bids -- code:1 rm: cannot remove '/scratch/workdir/62458e9fd3c0742103cbe7bd/bids/{projectname}/{participantlabel}excluded/.nfs00000000082800880000000a': Device or resource busy`

**Comment by dlevitas** [comment]
@soichih this seems like a server-side issue

**Comment by dlevitas** [comment]
@andreifoldes was this resolved in #49 or is this still an issue for?

**Comment by andreifoldes** [comment]
Yes I got my data in the end so we could say this is mute as well!
